### PR TITLE
feature: implement cli module

### PR DESCRIPTION
**Pull request by Yihen-Liu**
Signed-off-by: liuyihen@onchain.com

### CODE REVIEW

**Review by Arbio5zt — CHANGES_REQUESTED** [review]
1. cli seems no need to use actor.
2. could you redesign the `cmd helps` display formates. such as display in levels.

File: cmd/cmd.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+	"github.com/ontio/ontology/common/config"
+	"github.com/urfave/cli"
+)
+
+var ontSdk *sdk.OntologySdk
+
+func rpcAddress() string {
+	//return "http://139.219.108.204:20336"
+	index := rand.Intn(len(config.Parameters.SeedList))
+	return "http://" + (strings.Split(config.Parameters.SeedList[index], ":"))[0] + ":" + strconv.Itoa(config.Parameters.HttpJsonPort)
```
Comment (laizy) [inline cmd/cmd.go]: need use localhost ip , instead of connecting a remote node.

**Review by laizy — CHANGES_REQUESTED** [review]

File: cmd/contract_cmd.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+	if err != nil {
+		log.Fatalf("InvokeSmartContract WaitForGenerateBlock error:%s", err)
+	}
+	return nil
+}
+
+func getVmType(vmType uint) types.VmType {
+	switch vmType {
+	case 0:
+		return types.Native
```
Comment (Arbio5zt) [inline cmd/contract_cmd.go]: there has no way to deploy native contract, could please remove it ,thanks.

**Review by Arbio5zt — CHANGES_REQUESTED** [review]

File: cmd/utils/flags.go [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+	ConsensusFlag = cli.StringFlag{
+		Name:  "consensus",
+		Usage: "consensus turn on/off",
+	}
+
+	//contract deploy
+	ContractVmTypeFlag = cli.UintFlag{
+		Name:  "type",
+		Usage: "contract type ,value: NEOVM | NATIVE | SWAM",
```
Comment (laizy) [inline cmd/utils/flags.go]: fix this

**Review by laizy — CHANGES_REQUESTED** [review]

File: cmd/cmd.go (hunk 1 above)
Comment (Yihen-Liu) [reply cmd/cmd.go]: Thx. Have fix

File: cmd/contract_cmd.go (hunk 2 above)
Comment (Yihen-Liu) [reply cmd/contract_cmd.go]: Thx. have removed.

File: cmd/utils/flags.go (hunk 3 above)
Comment (Yihen-Liu) [reply cmd/utils/flags.go]: Thx. have fix

**Review by laizy — APPROVED** [review]

**Pull request merged**